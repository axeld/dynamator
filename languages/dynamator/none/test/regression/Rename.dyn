<dynamator language="none">
  <!-- Test rename for tag and attribute -->
  <tag tag="tag">
    <rename to="my:tag"/>
    <attr name="attr">
      <rename to="my:attr"/>
    </attr>
  </tag>
  <id name="id">
    <attr name="attr">
      <rename to="another:attr"/>
    </attr>
  </id>
</dynamator>
