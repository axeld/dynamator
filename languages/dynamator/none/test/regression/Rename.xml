<xml>
  <tag1 attr="val1">
    <tag/>
    <tag attr="val"/>
    <tag attr="val2" z="1">abc</tag>
  </tag1>
  <tag2 id="id" attr="val3"/>
</xml>
