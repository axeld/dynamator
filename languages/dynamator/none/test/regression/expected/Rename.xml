<xml>
  <tag1 attr="val1">
    <my:tag></my:tag>
    <my:tag my:attr="val"></my:tag>
    <my:tag my:attr="val2" z="1">abc</my:tag>
  </tag1>
  <tag2 id="id" another:attr="val3"></tag2>
</xml>
<!-- generated by Dynamator Sat Aug 07 12:43:50 CDT 2004 -->
